tests/grid/global_ids_01.cc

```diff
@@ -45,7 +45,7 @@ test(int n_refinements, MPI_Comm comm)
       deallog << cell->id() << " -> " << cell->subdomain_id() << " "
               << cell->global_active_cell_index() << std::endl;
 
-  const auto &part = tria.global_active_cell_index_partitioner();
+  const auto &part = *tria.global_active_cell_index_partitioner().lock();
```

Comment:
Here and in the other place below, I had to look up what `lock()` returns to be able to tell what the type of `part` should be. I would just be explicit instead of using `auto` to make that easier to read.

Reply:
Done :)

Reply:
You should be off work at this time of day!